GVFS/GVFS/RepairJobs/GitRefsRepairJob.cs

```diff
@@ -0,0 +1,163 @@
+﻿using GVFS.Common;
+using GVFS.Common.Git;
+using GVFS.Common.Tracing;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace GVFS.RepairJobs
+{
+    public abstract class GitRefsRepairJob : RepairJob
+    {
+        protected GitRefsRepairJob(ITracer tracer, TextWriter output, GVFSEnlistment enlistment) 
+            : base(tracer, output, enlistment)
+        {
+        }
+
+        public override IssueType HasIssue(List<string> messages)
+        {
+            int numBadRefs = 0;
+
+            foreach (string @ref in this.GetRefs())

```

Comment:
What is the `@` for?

Reply:
`ref` is a C# keyword. Prefixing with @ escapes the keyword. Could alternatively pick a different name for this variable? `reference` perhaps?

Reply:
I would go with `gitRef`, to avoid any chance of ambiguity